### PR TITLE
chore: Update README.md

### PR DESCRIPTION
**Pull request by klochowicz**
- mention how to use `homebrew`
- mention existence of the desktop app
- update bitcoin node distros support list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,27 +11,54 @@ Here is some material to read up on DLCs and ItchySats' protocol implementation:
...
+ItchySats is available on many DIY Bitcoin/Lightning node distributions:
+
+- [RaspiBlitz](https://github.com/rootzoll/raspiblitz#feature-overview)
+- [Umbrel](https://getumbrel.com/)
+- [Citadel](https://runcitadel.space/)
+
+### Desktop App
+
+We package ItchySats as an Electron app for Windows and MacOS.
+Latest release can be found [here](https://github.com/itchysats/itchysats/releases).
```
Comment (bonomat) [inline README.md]: You could link to latest: `https://github.com/itchysats/itchysats/releases/latest`

File: README.md [hunk 2]
```diff
@@ -11,27 +11,54 @@ Here is some material to read up on DLCs and ItchySats' protocol implementation:
...
+### Homebrew (macOS)
+
+We maintain a [homebrew](https://brew.sh/) tap with the latest version of ItchySats.
+
+```sh
+brew tap itchysats/itchysats
+brew install itchysats
+```
+
+After initial install, homebrew will take care of keeping ItchySats up-to-date.
```
Comment (bonomat) [inline README.md]: 🙃 

```suggestion
After the initial install, homebrew will take care of keeping ItchySats up-to-date.
```

File: README.md (hunk 1 above)
Comment (bonomat) [inline README.md]: 🙃
```suggestion
The latest release can be found [here](https://github.com/itchysats/itchysats/releases).
```

File: README.md [hunk 3]
```diff
@@ -11,27 +11,54 @@ Here is some material to read up on DLCs and ItchySats' protocol implementation:
...
+
+### Mobile
+
+ItchySats is not available as a full mobile app yet.
+In the meantime, if you'd like to use ItchySats on-the-go, the best way is to install ItchySats on your bitcoin node and use
+[Tailscale](https://tailscale.com/) or connect to your node via Tor.
```
Comment (bonomat) [inline README.md:44]: I know we have it somewhere else but personally I don't feel confident with recommending a service I have not evaluated. If you would trust it, I'm happy to leave it here though.

**Review by bonomat — APPROVED** [review]
Thanks for writing this up.

File: README.md [hunk 4]
```diff
@@ -11,27 +11,54 @@ Here is some material to read up on DLCs and ItchySats' protocol implementation:
...
 To open a position:
 
-1. Install the App on Umbrel (or start the binary / container)
-2. Transfer funds into the ItchySats wallet
-3. Open a `long` or `short` position - this will lock the funds in a multisig on chain
-4. Wait for the price to move
-5. Close the position - this will spend the locked up funds according to the price
+1. Open ItchySats
+2. Unlock ItchySats by typing in the default password (`weareallsatoshi` - except for Umbrel, where it depends on its seed and is visible in the AppStore)
+3. Choose your own secure password. Don't forget to store it securely; there's no way to reset it without removing ItchySats database.
```
Comment (bonomat) [inline README.md]: 🙃 The other bullet points have no `.`
```suggestion
3. Choose your own secure password. Don't forget to store it securely; there's no way to reset it without removing ItchySats database
```

**Review by bonomat — COMMENTED** [review]
Thanks for writing this up.

File: README.md [hunk 5]
```diff
@@ -11,27 +11,54 @@ Here is some material to read up on DLCs and ItchySats' protocol implementation:
...
+ItchySats is available on many DIY Bitcoin/Lightning node distributions:
+
+- [RaspiBlitz](https://github.com/rootzoll/raspiblitz#feature-overview)
+- [Umbrel](https://getumbrel.com/)
+- [Citadel](https://runcitadel.space/)
+
+### Desktop App
+
+We package ItchySats as an Electron app for Windows and MacOS.
+Latest release can be found [here](https://github.com/itchysats/itchysats/releases/latest).
```
Comment (bonomat) [inline README.md]:
```suggestion
The latest release can be found [here](https://github.com/itchysats/itchysats/releases/latest).
```

File: README.md (hunk 3 above)
Comment (klochowicz) [reply README.md:44]: I totally recommend evaluating it, the experience is way better than over Tor.
I use it myself, and also heard this being recommended as the go-to way to connect to your umbrel node by other members of the community, including Specter wallet dev at his talk at the latest conference.

File: README.md (hunk 4 above)
Comment (klochowicz) [reply README.md]: Thanks for pointing this out. They all should have `.`, as they start with the capital letter. changed in all of them.

File: README.md [hunk 6]
```diff
@@ -11,27 +11,54 @@ Here is some material to read up on DLCs and ItchySats' protocol implementation:
...
 To open a position:
 
-1. Install the App on Umbrel (or start the binary / container)
-2. Transfer funds into the ItchySats wallet
-3. Open a `long` or `short` position - this will lock the funds in a multisig on chain
-4. Wait for the price to move
-5. Close the position - this will spend the locked up funds according to the price
+1. Open ItchySats.
+2. Unlock ItchySats by typing in the default password (`weareallsatoshi` - except for Umbrel, where it depends on its seed and is visible in the AppStore).
+3. Choose your own secure password. Don't forget to store it securely; there's no way to reset it without removing ItchySats database.
```
Comment (luckysori) [inline README.md:57]: 🔧 
```suggestion
3. Choose your own secure password. Don't forget to store it securely; there's no way to reset it without removing the ItchySats database.
```

Comment (luckysori) [inline README.md]: 🔧 Missing word.
```suggestion
3. Choose your own secure password. Don't forget to store it securely; there's no way to reset it without removing the ItchySats database.
```

**Review by luckysori — APPROVED** [review]
Nice improvements!

**Pull request merged**